### ISSUE TITLE
Drag and drop a file into the yuzu app makes it crash

### ISSUE THREAD

**Issue by qt1b** [flathub/org.yuzu_emu.yuzu#260]
System info :

```
System:
  Host: qt1b Kernel: 5.13.19-2-MANJARO x86_64 bits: 64 compiler: gcc
    v: 11.1.0 Desktop: KDE Plasma 5.23.4 Distro: Manjaro Linux base: Arch Linux
Machine:
  Type: Desktop System: Gigabyte product: B450 I AORUS PRO WIFI v: N/A
    serial: <superuser required>
  Mobo: Gigabyte model: B450 I AORUS PRO WIFI-CF v: x.x
    serial: <superuser required> UEFI-[Legacy]: American Megatrends v: F50
    date: 11/27/2019
CPU:
  Info: 8-core model: AMD Ryzen 7 3700X bits: 64 type: MT MCP arch: Zen 2
    rev: 0 cache: L1: 512 KiB L2: 4 MiB L3: 32 MiB
  Speed (MHz): avg: 2296 high: 3592 min/max: 2200/4426 boost: enabled
    cores: 1: 3361 2: 2051 3: 2097 4: 1989 5: 1862 6: 1863 7: 1862 8: 2792
    9: 3592 10: 2392 11: 2053 12: 2056 13: 2192 14: 2196 15: 2192 16: 2193
    bogomips: 115050
  Flags: avx avx2 ht lm nx pae sse sse2 sse3 sse4_1 sse4_2 sse4a ssse3 svm
Graphics:
  Device-1: AMD Ellesmere [Radeon RX 470/480/570/570X/580/580X/590]
    vendor: Sapphire Limited Nitro+ driver: amdgpu v: kernel bus-ID: 0a:00.0
  Device-2: Microdia Dual Mode Camera (8006 VGA) type: USB
    driver: hid-generic,usbhid bus-ID: 3-1:2
  Display: x11 server: X.Org 1.21.1.2 driver: loaded: amdgpu,ati
    unloaded: modesetting resolution: 1920x1080
  OpenGL: renderer: AMD Radeon RX 580 Series (POLARIS10 DRM 3.41.0
    5.13.19-2-MANJARO LLVM 13.0.0)
    v: 4.6 Mesa 21.2.5 direct render: Yes
Network:
  Device-1: Intel Wireless-AC 9260 driver: iwlwifi v: kernel bus-ID: 07:00.0
  IF: wlp7s0 state: down mac: 42:be:02:c4:25:3e
  Device-2: Intel I211 Gigabit Network vendor: Gigabyte driver: igb
    v: kernel port: f000 bus-ID: 09:00.0
  IF: enp9s0 state: up speed: 100 Mbps duplex: full mac: 18:c0:4d:2e:34:cc
  Device-3: Intel Wireless-AC 9260 Bluetooth Adapter type: USB
    driver: btusb bus-ID: 1-10:3
  IF-ID-1: tun0 state: unknown speed: 10 Mbps duplex: full mac: N/A
Drives:
  Local Storage: total: 2.75 TiB used: 401.33 GiB (14.2%)
Info:
  Processes: 377 Uptime: 36m Memory: 31.3 GiB used: 3.48 GiB (11.1%)
  Init: systemd Compilers: gcc: 11.1.0 clang: 13.0.0 Packages: 1600
  Shell: fish v: 3.3.1 inxi: 3.3.11

```

logfile :
```
[ 301.793696] Frontend <Info> yuzu/main.cpp:BootGame:1363: yuzu starting...
[ 301.793810] Common.Filesystem <Error> common/fs/file.cpp:Open:261: Failed to open the file at path=/home/qt1b/Games/Nintendo/switch/vvvvv, ec_message=No such file or directory
[ 301.793818] Common.Filesystem <Error> common/fs/fs.cpp:FileOpen:150: Failed to open the file at path=/home/qt1b/Games/Nintendo/switch/vvvvv with mode=1, type=0, flag=1
```

It seems to be a flatpak permission issue, but it shouldn't make yuzu crash in normal circumstances 

Hope this will be helpful

**Comment by Revision18** [comment]
Yuzu should show an error dialog when this happens and no crash should happen. Also, it can link to flatpak help so people learn how to give permissions in flatpak.

I fixed a similar problem by running:

`flatpak override --user --filesystem=/media/Data org.yuzu_emu.yuzu`

Replace `/media/Data` with your problematic path.

**Comment by Sunderland93** [comment]
Same issue, but different message:
 ```
Unhandled SIGSEGV at rip 0x00558646c0edf5
/app/bin/yuzu-launcher: строка 18:     3 Segmentation fault                   prlimit --nofile=8192 yuzu "$@"
```

**Comment by liushuyu** [comment]
This should have been fixed in #315.

Please feel free to re-open this issue if the issue with drag and drop still persists.
